game/sprite-backend.rpy

```diff
@@ -1082,9 +1086,57 @@ image natsuki idle = ConditionSwitch(
     predict_all = True
 )
 
+image natsuki test = ConditionSwitch(
+    "Natsuki.getMouseIsLeft() and Natsuki.getMouseIsAbove()", "natsuki 5utlsml",
+    "Natsuki.getMouseIsRight() and Natsuki.getMouseIsAbove()", "natsuki 5utrsml",
+    "Natsuki.getMouseIsLeft() and Natsuki.getMouseIsBelow()", "natsuki 5udlsml",
+    "Natsuki.getMouseIsRight() and Natsuki.getMouseIsBelow()", "natsuki 5udrsml",
+    "Natsuki.getMouseIsLeft()", "natsuki 5ullsml",
+    "Natsuki.getMouseIsRight()", "natsuki 5ulrsml",
+    "Natsuki.getMouseIsAbove()", "natsuki 5uupsml",
+    "Natsuki.getMouseIsBelow()", "natsuki 5udwsml",
+    "True", "natsuki 5unmsml",
+    predict_all = True
+)
+
+label dynatest:
+    show natsuki test
+    n ""
+    return
+
```

Comment:
Is this leftover test code? If yes, we want to get rid of it!

Reply:
Indeed... yes...